app/views/admin/users/_navigation.html.erb

```diff
@@ -0,0 +1,9 @@
+<nav class="mb-2">
+  <ul class="crayons-navigation crayons-navigation--horizontal">
+    <li><a href="#" class="crayons-navigation__item crayons-navigation__item--current" aria-current="page">Overview</a></li>
+    <li><a href="#" class="crayons-navigation__item">Notes</a></li>
+    <li><a href="#" class="crayons-navigation__item">Emails</a></li>
+    <li><a href="#" class="crayons-navigation__item">Reports</a></li>
+    <li><a href="#" class="crayons-navigation__item">Flags</a></li>
```

Comment:
I need views for these tabs and appropriate URLs too obviously. Oh and some quick'n'easy way to mark tab as currently selected.

Reply:
Also, we'll look at this in a separate PR.